lib/image_info/parser.rb

```diff
@@ -3,15 +3,15 @@
 module ImageInfo
   class Parser
 
-    attr_reader :image, :bytes, :parser
+    attr_reader :image
 
     def initialize(image, data)
-      @image  = image
-      @bytes  = data.bytes
-      @parser = ::ImageSize.new(data)
+      @image = image
+      @data = data
```

Comment:
please make that a `attr_reader` and use `data` instead of `@data` in the class directly